"Could not find com.google.android:flexbox:2.0.1" during build

Gradle build fails:

```
./gradlew clean build
```
```
> Task :app:checkDebugAarMetadata FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.google.android:flexbox:2.0.1.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/google/android/flexbox/2.0.1/flexbox-2.0.1.pom
       - https://jcenter.bintray.com/com/google/android/flexbox/2.0.1/flexbox-2.0.1.pom
       - https://jitpack.io/com/google/android/flexbox/2.0.1/flexbox-2.0.1.pom
     Required by:
         project :app
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -57,7 +57,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
 
     implementation 'com.google.android.material:material:1.10.0'
-    implementation 'com.google.android:flexbox:2.0.1'
+    implementation 'com.google.android.flexbox:flexbox:3.0.0'
 
     testImplementation 'junit:junit:4.13.2'
 

